src/reducers/GraphDataState.ts

```diff
@@ -60,6 +75,50 @@ const graphDataState = (state: GraphState = INITIAL_GRAPH_STATE, action: KialiAp
       newState.graphDataTimestamp = INITIAL_GRAPH_STATE.graphDataTimestamp;
       newState.sidePanelInfo = INITIAL_GRAPH_STATE.sidePanelInfo;
       break;
+    // Filter actions
+    case getType(GraphFilterActions.setEdgelLabelMode):
+      newState.filterState.edgeLabelMode = action.payload;
+      break;
+    case getType(GraphFilterActions.setGraphType):
+      newState.filterState.graphType = action.payload;
+      // Init graph on graphType change
+      newState.graphData = INITIAL_GRAPH_STATE.graphData;
+      newState.graphDataTimestamp = INITIAL_GRAPH_STATE.graphDataTimestamp;
+      newState.sidePanelInfo = INITIAL_GRAPH_STATE.sidePanelInfo;
+      break;
+    case getType(GraphFilterActions.toggleLegend):
+      newState.filterState.showLegend = !state.filterState.showLegend;
+      break;
+    case getType(GraphFilterActions.toggleGraphNodeLabel):
+      newState.filterState.showNodeLabels = !state.filterState.showNodeLabels;
+      break;
+    case getType(GraphFilterActions.toggleGraphCircuitBreakers):
+      newState.filterState.showCircuitBreakers = !state.filterState.showCircuitBreakers;
+      break;
+    case getType(GraphFilterActions.toggleGraphVirtualServices):
+      newState.filterState.showVirtualServices = !state.filterState.showVirtualServices;
+      break;
+    case getType(GraphFilterActions.toggleGraphMissingSidecars):
+      newState.filterState.showMissingSidecars = !state.filterState.showMissingSidecars;
+      break;
+    case getType(GraphFilterActions.toggleGraphSecurity):
+      newState.filterState.showSecurity = !state.filterState.showSecurity;
+      break;
+    case getType(GraphFilterActions.toggleServiceNodes):
+      newState.filterState.showServiceNodes = !state.filterState.showServiceNodes;
+      // Init graph on serviceNodeschange
+      newState.graphData = INITIAL_GRAPH_STATE.graphData;
+      newState.graphDataTimestamp = INITIAL_GRAPH_STATE.graphDataTimestamp;
+      newState.sidePanelInfo = INITIAL_GRAPH_STATE.sidePanelInfo;
+      break;
+    case getType(GraphFilterActions.toggleTrafficAnimation):
+      newState.filterState.showTrafficAnimation = !state.filterState.showTrafficAnimation;
+      break;
+    case getType(GraphFilterActions.toggleUnusedNodes):
+      newState.filterState.showUnusedNodes = !state.filterState.showUnusedNodes;
+      break;
+    // case getType(GraphFilterActions.showGraphFilters):
+    //   return updateState(state, { disableLayers: action.payload });
```

Comment:
Is this left commented on purpose?

Reply:
It was there when I started this.  I can remove it though, I don't know why it's actually there.